### DISCUSSION TITLE
The V2 of Ticket Bot is here!

### DISCUSSION THREAD

**Discussion by Sayrix** [Sayrix/Ticket-Bot#69] [Announcements]
You can also join my discord server to ask me for help directly on Discord: [discord.gg/VasYV6MEJy](https://discord.gg/VasYV6MEJy)
I also created a very small documentation: [ticket-bot.pages.dev](https://ticket-bot.pages.dev)

**Comment by svenbledt** [comment]
What makes your bot different from others? Shouldn't be a front! Im rly interested to your work!

**Reply by Sayrix** [reply]
Hey! my Ticket-Bot have a lot of features, like a transcript system; discord buttons; discord slash commands; discord select menus; discord embeds; the possibility to change every embeds of the bot; a database system and a lot of others features, you really need to try the bot it takes really 5 minutes to config it and try it
